### DISCUSSION TITLE
“快速收起评论”功能消失

### DISCUSSION THREAD

**Discussion by Neo-Ze** [the1812/Bilibili-Evolved#3245] [提问 / Q&A] [answered]
### 如图
脚本版本：v2.1.7
浏览器版本：Chrome 99.0.4844.82（64 位）
![image](https://user-images.githubusercontent.com/81399833/163575071-dd1004c7-16a0-471e-b270-dd71ecf504f4.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
请
善
用
搜
索
#3191
